app/controllers/admin/members/base_controller.rb

```diff
@@ -13,6 +13,7 @@ def are_appointments_enabled?
 
       def load_member
         @member = Member.find(params[:member_id])
+        @notes = @member.notes.pinned_first
```

Comment:
As discussed this should probably be in a `load_notes` method

Reply:
Would look something like this:
```ruby
before_action :load_notes
def load_notes
  @notes = @member.notes.pinned_first
end
```

Reply:
I believe I made that change in my latest commit. just wanted to confirm if it usually updates here automatically?

Reply:
@fgonza23 Looks like this change didn't make it through. Could you give it another try?